### PR TITLE
Update DISA STIG related files to newest version V2R4

### PR DESCRIPTION
**Pull request by ggbecker**
#### Description:

- Update DISA STIG related files to newest version V2R4

- Source: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux

- Relates to #4685 ?

Additional info:
I've made these changes based on some guides and history of commits, I'm not sure if it's missing something or if they are really wanted at this time, but I saw outdated content and changed what I could. This PR does not touch the rules which could be updated by this newer version of DISA STIG.

### CODE REVIEW

**Comment by ggbecker** [comment]
@yuumasato is this a valid PR?

File: utils/create-stig-overlay.py [hunk 1]
```diff
@@ -159,11 +159,11 @@ def main():
         ssg_xccdftree = ET.parse(args.ssg_xccdf_filename)
         ssg = ssg_xccdftree.find(".//{%s}publisher" % dc_ns).text
         if ssg != "SCAP Security Guide Project":
-            sys.exit("%s is not a valid SSG generated XCCDF file." % ssg_xccdf_filename)
+            sys.exit("%s is not a valid SSG generated XCCDF file." % args.ssg_xccdf_filename)
```
[the remark this answers is not in the document]
Comment (ggbecker) [reply utils/create-stig-overlay.py]: The variable `ssg_xccdf_filename` is undefined at this point. See line `159`, the correct is `args.ssg_xccdf_filename`.

**Comment by yuumasato** [comment]
> @yuumasato is this a valid PR?

LGTM.

> Should we also update files for RHEL6, from V1R21 to V1R23?

If have cycles yes. Then please do in separate PR.

**Comment by yuumasato** [comment]
> @yuumasato is this a valid PR?

@shawndwells @redhatrises Do you have any remarks? Thanks

**Comment by redhatrises** [comment]
> > @yuumasato is this a valid PR?
> 
> @shawndwells @redhatrises Do you have any remarks? Thanks

Didn't realize you were waiting on us. Ack.

**Pull request merged**